torch/utils/mkldnn.py

```diff
@@ -59,13 +59,15 @@ def __getstate__(self):
 
     @torch.jit.script_method
     def forward(self, x):
-        return torch.mkldnn_convolution(
+        return torch.convolution(
```

Comment:
enable it at https://github.com/pytorch/pytorch/pull/40483

Reply:
what is convolution behavior wrt mkl or dense inputs? Similar to cases you posted above for linear and bn.

Reply:
After #40610 torch.convolution won't be calling mkldnn in a fair number of cases, so you still need to call mkldnn_convolution here.

Reply:
@ngimel, I don't think need to call mkldnn_convolution here, if input and weight are MKLDNN tensor, it will call mkldnn_convolution directly, see https://github.com/pytorch/pytorch/blob/fcadca1bdab62ea5728e49397e023c8863466e83/aten/src/ATen/native/Convolution.cpp#L230-L239. if users set input to MKLDNN tensor, assuming they want to call MKLDNN path.

Reply:
@ngimel